docs/containers.conf.5.md

```diff
@@ -567,6 +567,11 @@ not be by other drivers.
 Determines whether file copied into a container will have changed ownership to
 the primary uid/gid of the container.
 
+**compression_format**=""
+
+Specifies the compression format to use when pushing an image.  Supported values are: `gzip`, `zstd` and `zstd:chunked`.  The default is "".
```

Comment:
```suggestion
Specifies the compression format to use when pushing an image.  Supported values are: `gzip`, `zstd` and `zstd:chunked`.  The default is "gzip".
```

Reply:
> Isn't the default "gzip" or are you relying on the containers/image. Saying the default is "" gives the user no information.

Good point.  Probably not mention a default at all.

Reply:
dropped the sentence `The default is "".`